src/shared/components/common/user-badges.tsx

```diff
@@ -43,13 +42,24 @@ function getRoleLabelPill({
 @tippyMixin
 export class UserBadges extends Component<UserBadgesProps> {
   render() {
+    const isBot = this.props.person?.bot_account;
+    const isDeleted = this.props.person?.deleted;
+    var isNewAccount = false;
+    if (this.props.person !== undefined) {
+      // 7 days ago
+      // https://stackoverflow.com/a/563442
+      var date = new Date();
+      date.setDate(date.getDate() - 7);
+      isNewAccount = new Date(this.props.person?.published_at) > date;
+    }
```

Comment:
Simplified this by passing only `Person`, instead of passing various person fields separately (is bot, is deleted, published date).